src/common-principles.md

```diff
@@ -430,36 +430,54 @@ NIfTI header.
 
 ### Tabular files
 
-Tabular data MUST be saved as tab delimited values (`.tsv`) files, that is, CSV
-files where commas are replaced by tabs. Tabs MUST be true tab characters and
-MUST NOT be a series of space characters. Each TSV file MUST start with a header
-line listing the names of all columns (with the exception of
-[physiological and other continuous recordings](modality-specific-files/physiological-and-other-continuous-recordings.md)
-as well as [motion recording data](modality-specific-files/motion.md)).
+Tabular data MUST be saved as plain-text, tab-delimited values (TSV) files
+(with [extension `.tsv`](glossary.md#tsv-extensions)),
+that is, CSV files where commas are replaced by tab characters.
```

Comment:
this reminded me a (possibly an "incorrect") anecdote about two color blind folks.  To add to the insult -- I do not remember it truly ;)  But here it goes for the same meaning:

One color blind person asks another :
- have you seen my new green car?
- what kind of color is that "green"?
- have you seen the red sunset?
- yes
- so it is like that color but just "green".

We never talk about CSV files in the specification and there is an assumption that users know what those are, and we do not even have a URL.  Let's may be at least add a link if to keep it at all?

```suggestion
(with [extension `.tsv`](glossary.md#tsv-extensions)),
that is, [CSV files](https://en.wikipedia.org/wiki/Comma-separated_values) where commas are replaced by tab characters.
```